device/bgp/bgpv6srtepolicy.yaml

```diff
@@ -26,20 +22,12 @@ components:
             destination prefixes to steer traffic into the SR Policy. 
           type: integer
           default: 100
-        ipv4_endpoint:
-          description: >-
-            Specifies a single node or a set of nodes.
-            It is selected on the basis of the policy_type (AFI).
-          type: string
-          format: ipv4
-          default: 0.0.0.0
         ipv6_endpoint:
           description: >-
             Specifies a single node or a set of nodes.
```

Comment:
Should anycast address be mentioned here ?

Reply:
corrected the description for IPv6 Endpoint.